### ISSUE TITLE
Mock commands?

### ISSUE THREAD

**Issue by vspinu** [rejeep/el-mock.el#4]
Hi, 

Is there a way to mock interactive commands?

```
(mocklet ((test1 => (interactive) (message "test1")))
  (call-interactively 'test1))
```

**Comment by rejeep** [comment]
No idea actually, I never use `mocklet` myself. I don't know the el-mock code that well myself. I'm only hosting it to avoid the Emacs wiki.

Let us know if you find out! :)

**Comment by vspinu** [comment]
> No idea actually, I never use mocklet myself. I don't know the el-mock code that well myself. I'm only hosting it to avoid the Emacs wiki.

That's funny.

Then you should probably keep the issue open till someone could fix it. Or is this project completely dead now?

**Comment by rejeep** [comment]
No, it's not dead. If a serious bug is found, I will just have to dig into the code and fix it.

You are right, I'm leaving it open!

**Comment by telotortium** [comment]
@vspinu Are you trying to confirm that your mock was called interactively, or just trying to set up a mock that *can* be called interactively?

**Comment by telotortium** [comment]
Maybe #26 (unmerged as of now) may be able to address your use case.
